socorro/signature/utils.py

```diff
@@ -36,7 +36,7 @@ def convert_to_crash_data(raw_crash, processed_crash):
 
     crash_data = {
         # JavaStackTrace or None
-        'java_stack_trace': glom(raw_crash, 'JavaStackTrace', default=None),
+        'java_stack_trace': glom(processed_crash, 'java_stack_trace', default=None),
```

Comment:
Need to change signature generation to get the processed crash version since that'll be sanitized.

Also, this has the nicety of reducing variability in signatures. That might fix 1367027 and 982233 or at least alleviate them.